packages/terra-html-table/src/Table.module.scss

```diff
@@ -94,14 +99,6 @@
         }
       }
     }
-
-    tfoot {
-      tr {
-        border-bottom: var(--terra-html-table-footer-row-border-bottom, 1px solid #dedfe0);
-        border-top: var(--terra-html-table-footer-row-border-top, 0);
-        text-align: left;
-      }
-    }
```

Comment:
Why was this is removed?

Reply:
Currently, html-table does not use any className as tfoot. So I thought of removing the extra styles, which are not being consumed.